portfolio/src/main/webapp/script.js

```diff
@@ -28,12 +28,34 @@ const hideMenuShowMain = () => {
   }
 }
 
- /**
-  * ADD EVENT LISTENERS
-  */
+/* HOME */
+const actions = document.getElementsByClassName('action');
```

Comment:
You don't need to put all logic together into one js file, you can have multiple js file where each handles one part

Reply:
Thanks for the suggestion! I didn't separate them because the file isn't that huge. Will do so though.